README.md

```diff
@@ -18,7 +20,7 @@ for more background, objectives, key use cases.
 
 ### Requirements
 
-* [Node](https://nodejs.org/) 9 (with npm)
+* [Node](https://nodejs.org/) 8+ (with npm)
```

Comment:
Our Dockerfile specifies Node 8, so I modified this. Plus, the 9.x branch isn't an LTS channel, so ... yeah.